sdk/search/Microsoft.Azure.Search/tests/Tests/Models/CustomAuthorConverter.cs

```diff
@@ -64,7 +64,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
         {
             var author = (TAuthor)value;
 
-            string[] names = author.FullName.Split(' ', StringSplitOptions.RemoveEmptyEntries);
+            string[] names = author.FullName.Split(new char[] {' '}, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Why was this change required?

Reply:
This Split overload doesn't exist in .NET 4.6.1 (only on .NET Core) so we just needed to use the other overload.